imageglass.json

```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "http://www.imageglass.org/",
+    "description": "A lightweight, versatile image viewer.",
+    "license": "https://github.com/d2phap/ImageGlass/blob/master/LICENSE",
+    "version": "4.5.11.27",
+    "url": "https://github.com/d2phap/ImageGlass/releases/download/4.5.11.27/ImageGlass_4.5.11.27.zip",
+    "hash": "6e518b82b2d9521ec977a66c94a756f304f430c6332634280d02ca554cb0940b",
+    "extract_dir": "ImageGlass_4.5.11.27",
+    "persist": [
+        "igconfig.xml"
+    ],
+    "bin": [
+        [
+            "ImageGlass.exe",
+            "imageglass"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "ImageGlass.exe",
+            "imageglass"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/d2phap/ImageGlass/"
+    },
+    "autoupdate": {
+        "url": "https://github.com/d2phap/ImageGlass/releases/download/4.5.11.27/ImageGlass_$version.zip"
```

Comment:
Need to $version the other 4.5.11.27.

Also, it would be helpful to add an `autoupdate` section.

Reply:
@yigitemres yes, you also have to add the `extract_dir` property to `autoupdate`
```json
    "autoupdate": {
        "url": "https://github.com/d2phap/ImageGlass/releases/download/$version/ImageGlass_$version.zip",
        "extract_dir": "ImageGlass_$version"
    }
```